test/helpers.bash

```diff
@@ -304,9 +303,9 @@ function setup_crio() {
         --listen "$CRIO_SOCKET" \
         --registry "quay.io" \
         --registry "docker.io" \
-        --runtimes "$RUNTIME_NAME:$RUNTIME_BINARY:$RUNTIME_ROOT:$RUNTIME_TYPE" \
         -r "$TESTDIR/crio" \
         --runroot "$TESTDIR/crio-run" \
+        --runtimes "$CONTAINER_RUNTIMES" \
```

Comment:
If we export CONTAINER_RUNTIMES can we also skip the --runtimes parameter here?

Reply:
I found some weirdness without it, but I forget what that was. Let's try!